pwiz_tools/Skyline/Util/SSRCalc3.cs

```diff
@@ -1046,9 +1046,9 @@ private double helicity1(string sq)
                 {
                     hc6 = hc.Substring(i, 6);
                     sc6 = 0.0;
-                    if (HlxScore6.ContainsKey(hc6))
+                    if (HlxScore6.TryGetValue(hc6, out var value))

```

Comment:
This if block could be simplified to:
`HlxScore6.TryGetValue(hc6, out sc6);`
because TryGetValue will set the out parameter to zero if it returns false

Reply:
Fair enough, there are a couple more like it in there I'll deal with as well.